game.sh

```diff
@@ -2,3 +2,9 @@ echo "Hello from `whoami`"
 echo "Terminal time:`date`"
 echo "This line should be here"
 echo "Spider locomotion"
+
```

Comment:
Hey, you introduced a lot empty new lines. Can you fix it please.

Reply:
On it boss